app/src/main/res/drawable/action_bookmark.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="36dp"
+    android:height="36dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0">
+    <path
```

Comment:
2 space tabs for XML please in all locations.

Reply:
Done!